test/testhelpers.cpp

```diff
@@ -0,0 +1,11 @@
+#include <unistd.h>
+#include "testhelpers.h"
+#include "../src/tools.h"
+
+
+TestHelpers::TestHelpers()
```

Comment:
I dont see the need for adding this class. Please use a helper function if nedded.

Reply:
Indeed. I thought I need more flexibility, but I didn't.

Reply:
class and files testhelpers.(cpp|h) removed.